packages/paste-website/src/pages/patterns/data-export/index.mdx

```diff
@@ -0,0 +1,264 @@
+  <Column>
+    <Card>
+      <Heading as="h3" variant="heading40" marginBottom="space0">
+        <Anchor href="/icons">Export Icon</Anchor>
+      </Heading>
+    </Card>
+  </Column>
+  <Column>
+    <Card>
+      <Heading as="h3" variant="heading40" marginBottom="space0">
+        <Anchor href="/components/toast">Toast</Anchor>
+      </Heading>
+    </Card>
+  </Column>
+  <Column>
+    <Card>
+      <Heading as="h3" variant="heading40" marginBottom="space0">
+        <Anchor href="/components/tooltip">Tooltip</Anchor>
+      </Heading>
+    </Card>
+  </Column>
+</Grid>
+
+```jsx
+// import all ingredients for the Data export pattern
+
+import {​ Button } from "@twilio-paste/core/button";
+import { ExportIcon } from "@twilio-paste/icons/esm/ExportIcon";
+import { Menu } from "@twilio-paste/core/menu";
+import { Tooltip } from "@twilio-paste/core/tooltip";
+import { Anchor } from "@twilio-paste/core/anchor";
+import { Toast } from "@twilio-paste/core/toast";
+```
+
+## Usage
+
+### General
+
+The Data export pattern enables customers to extract all or selected data
+from a data table or graph. Data is usually placed into a CSV, PNG, PDF,
+or JSON file. In general, data exports are generated by the server in
+Twilio products (particularly CSVs in Voice & Video, SIP trunking, and Messaging).
+
+Exporting should:
+
+- Be triggered by a [Button](/components/button) (or [Button as Anchor](/patterns/button-vs-anchor#buttons-with-link-functionality))
+  or [Menu item](/components/menu). The Button or Menu item should clearly state what type of file
+  will be exported. E.g. “Export CSV”.
+- Tell the customer if there are any limitations to the data that can be exported.
+  Use a [Tooltip](/components/tooltip) to communicate this information.
+
+<LivePreview scope={{Tooltip, Button, ExportIcon}} noInline language="jsx">
+  {exportButton}
+</LivePreview>
+
+Using the word “Export” and the ExportIcon are predictable signals to customers that they can extract data from Twilio.
+
+Twilio products generally enable customers to convert files to a range of formats before downloading and saving. However if you can’t provide a format conversion, you should still use the word “Export” and the ExportIcon.
+
+The delayed export is an exception to this copy guideline and uses the word “Download” and the DownloadIcon instead. action is placed at the top right of the area where the object will appear once it has been created. In this example, the entire page is dedicated to displaying the user’s services, so the “create new service” action is placed at the top of the page next to the page header.
+
+## Variations
+
+### Immediate export
+
+An immediate export happens when a customer can download their export on
+the same page where they clicked the export Button or Menu item.
+
+When using a Button, while an export is in progress, show the Button loading
+state. When it’s in a loading state, it will disable by default.
+
+<LivePreview
```

Comment:
Could this live preview include both the regular export button (e.g. Export CSV) and the menu export button (like you already have here)?

Reply:
Can you also make the regular export button show the loading state for 2 seconds on click? And then revert to the default state.